content/yaml-running-builds/starting-builds-automatically.md

```diff
@@ -21,6 +21,18 @@ For repositories added via SSH or HTTP/HTTPS, or if you are configuring your bui
 
 In the `events:` section, specify which events in the repository trigger builds.
 - **push** - a build will be started every time you commit code to any of the tracked branches.
+
+    GitHub Merge Queue as a feature helps manage and automate the process of merging multiple pull requests (PRs) into a branch. Merge Queue allows to automate and serialize merges, making sure that every pull request is tested not in isolation, but as it will be after it’s merged. More information can be found [here](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue). 
+    
+    To configure it with Codemagic workflows, a branch name prefixed with *gh-readonly-queue/{base_branch}* needs to be added in **codemagic.yaml** as explained [here](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#triggering-merge-group-checks-with-third-party-ci-providers)
```

Comment:
```suggestion
To configure it with Codemagic workflows, a branch name prefixed with *gh-readonly-queue/{base_branch}* needs to be added in **codemagic.yaml** as explained [here](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#triggering-merge-group-checks-with-third-party-ci-providers)
```

Reply:
If that space is removed, it will stay on the same line as `push`. Here is what it looks like now:
<img width="830" height="442" alt="Screenshot 2025-09-05 at 3 52 39 PM" src="https://github.com/user-attachments/assets/538a0a06-b496-4350-8dc7-bb259908a8df" />